### PR TITLE
Sync'ed the IDL with latest hr-time

### PR DESCRIPTION
**Pull request by plehegar**
... except for translateTime since it's too early.

### CODE REVIEW

File: hr-time/idlharness.html [hunk 1]
```diff
@@ -15,24 +15,41 @@
 <div id="log"></div>
 
 <pre id='untested_idl' style='display:none'>
-
 interface Window {
 };
 
-interface Performance {
+[Constructor(DOMString scriptURL), Exposed=(Window,Worker)]
```
Comment (zqzhang) [inline hr-time/idlharness.html]: Seems the testing framework doesn't handle `Exposed=(Window,Worker)` properly.

Existing tests in this repository either commented it out, or split it into windows and worker tests (#1989).

**Comment by gsnedders** [comment]
```
15:01 < aryehgregor> Um, I might have done something wrong by mistake.
15:01 < aryehgregor> I think I pushed to the wrong repo?
```

**Comment by ayg** [comment]
I can't figure out how to restore the branch at this point, so please re-push it if you still have it locally in case I don't wind up finding a solution.

**Comment by plehegar** [comment]
This should be closed. It has been resolved by https://github.com/w3c/web-platform-tests/pull/4069.

**Pull request closed without merging**